package.json

```diff
@@ -51,13 +51,17 @@
     "@types/mocha": "^8.0.3",
     "@types/node": "^14.14.2",
     "@types/puppeteer": "~3.0.2",
+    "@typescript-eslint/eslint-plugin": "^4.5.0",
+    "@typescript-eslint/parser": "^4.5.0",
     "basic-auth": "^2.0.1",
     "eslint": "^7.12.0",
+    "eslint-plugin-import": "^2.22.1",
```

Comment:
I'm not completely sure what this plugin is for (in this project) as no options from the plugin docs are used in the eslint.

Reply:
good point. it was automatically added by `tslint-to-eslint-config` so i'd assumed the project must need it, but i've removed the package and linting still passes 👍

Reply:
Cool, thanks for checking